node-zerox/src/index.ts

```diff
@@ -21,6 +21,9 @@ export const zerox = async ({
   openaiAPIKey = "",
```

Comment:
We should also add this quickly to the python package to maintain as much feature parity.